lib/web_error.dart

```diff
@@ -0,0 +1,72 @@
+import 'error.dart';
+import 'js_agent_interop.dart';
+
+FingerprintProError unwrapWebError(WebException error) {
+  final message = error.message;
+
+  if (message == FingerprintJS.ERROR_NETWORK_CONNECTION) {
```

Comment:
Can be replace this with a switch statement?

Reply:
No, because of interop with js. Dart allows only constants.